kissmp-bridge/src/voice_chat.rs

```diff
@@ -1,8 +1,23 @@
+/*
+    Remarks regarding cpal:
```

Comment:
This comment is useless and provide no useful information. It should be obvious for a person why blocking MPSC is used here, also, your explanation of why it's used is incorrect.
First of all, CPAL needs multiple threads. Second, we're using normal mpsc receiver just because we're using it in non-async context, sender however is still async.